conda_smithy/linter/hints.py

```diff
@@ -266,21 +266,26 @@ def hint_noarch_python_use_python_min(
                     break
             else:
                 hint += (
-                    f"For the `{section_name}` section of the recipe, you should almost always use `{syntax}` "
-                    f"for the `python` entry. "
+                    f"\n  - For the `{section_name}` section of the recipe, you should usually use `{syntax}` "
+                    f"for the `python` entry."
                 )
 
         if hint:
             hint = (
                 (
-                    "noarch: python recipes should almost always follow the syntax in "
-                    "our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python). "
+                    "`noarch: python` recipes should usually follow the syntax in "
```

Comment:
I was thinking more something like this:

```suggestion
                    "**Experimental:** (A migration for this is planned for the near future.) `noarch: python` recipes should usually follow the syntax in "
```

Since they usually indicate some form of typo, I personally don't think of the linter "suggestions" / "hints" as optional, even though they technically are. So I'm trying to emphasize here more something like, "You don't need to do this now, but we'll be automatically doing most of this for you soon."

Reply:
Thanks so much @beckermr, I really appreciate your diligence!!

Reply:
Yeah this is not experimental though. We passed a CFEP and this hint will remain in place even after the migration.

Reply:
Ya, "experimental" was not the best choice of words.

Although verbose, perhaps `**Safe to temporarily ignore:**`?

Reply:
But one can always ignore hints. I am not quite sure what you are going for here. We've had some linter bugs and made changes to clarify things, but otherwise the hint is correct.

Reply:
What I'm going for is to try and prevent casual recipe maintainers from needing to read CFEP-25.

> But one can always ignore hints.

Ok, sure, it's definitely partially my fault that I wasn't reading carefully enough. I didn't read anything from the linter's message above "For **recipe/meta.yaml**:" since it's mostly boilerplate, and thus I didn't realize. This linter is very quiet (which is very good!), so whenever I see something, I think that I messed something up and action is required.

Due to my inattentive reading I ended up going down a much bigger rabbit hole than was necessary, so I'm looking to preempt that for other maintainers.

But everything in the message is technically correct, so it's fine as-is.

Reply:
Ok thanks for working with me! Resolving.